test/e2e/common_test.go

```diff
@@ -937,3 +1009,32 @@ func writeYaml(content string, fileName string) error {
 
 	return nil
 }
+func GetPort() int {
+	a, err := net.ResolveTCPAddr("tcp", "localhost:0")
+	if err != nil {
+		Expect("").To(Equal("Unable to find port"))
```

Comment:
I don't know what circumstances will cause this to fail, but maybe when it does, the person investigating would appreciate seeing some trace of what `err` is? (line 1020 also)